### PR TITLE
Update github issue templates

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

**Comment by cybette** [comment]
lgtm :+1:

File: .github/ISSUE_TEMPLATE/submit-a-talk.md [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
 about: Tell us about a talk you'd like to give
+body:
+- type: markdown
+  attributes:
+    value: >
+      Thank you for your interest in talking at an Ansible community meetup !
+      We are looking forward to your proposal on a wide range of Ansible topics whether you are a user, a contributor or a developer.
 
----
+      If you have any questions, please feel free to reach out to us in #community:ansible.com on Matrix or #ansible-community on [libera.chat](https://libera.chat/).
```
Comment (gundalow) [inline .github/ISSUE_TEMPLATE/submit-a-talk.md]:
```suggestion
      If you have any questions, please feel free to reach out to us via:
      * Matrix: [#community:ansible.com](https://matrix.to/#/#community:ansible.im)
      * Libera.chat (IRC):` #ansible-community` on [libera.chat](https://web.libera.chat/#libera)
```

File: .github/ISSUE_TEMPLATE/submit-an-idea.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+name: Submit an idea
+about: Tell us about a talk you'd like to see
+body:
+- type: markdown
+  attributes:
+    value: >
+      Thank you for your interest in the Ansible community meetups !
+      We are looking forward to your proposal on a wide range of Ansible topics whether you are a user, a contributor or a developer.
+
+      If you have any questions, please feel free to reach out to us in #community:ansible.com on Matrix or #ansible-community on [libera.chat](https://libera.chat/).
```
Comment (gundalow) [inline .github/ISSUE_TEMPLATE/submit-an-idea.md]:
```suggestion
      If you have any questions, please feel free to reach out to us via:
      * Matrix: [#community:ansible.com](https://matrix.to/#/#community:ansible.im)
      * Libera.chat (IRC):` #ansible-community` on [libera.chat](https://web.libera.chat/#libera)
```

**Comment by dmsimard** [comment]
Thanks @gundalow and @cybette !

**Pull request merged**